### DISCUSSION TITLE
Does deepgram provides a redaction feature for call audio file and output redacted audio file?

### DISCUSSION THREAD

**Discussion by Lokesh-TCP** [deepgram/community#708] [General help] [answered]
In my use case,
I have a call audio file with PII and PCI. I want to Redacts these info and save the output in audio file. ( not transcript )

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@Lokesh-TCP  We currently don't offer a redaction feature that would output the redacted audio into a new audio file.  You can see what we currently have available for speech to text  Redaction here : https://developers.deepgram.com/docs/redaction

One idea: Is you could use or [text to speech API ](https://developers.deepgram.com/docs/text-to-speech)to take the redacted transcript text from an audio filr and send that to text be converted to audio using one of our [Voice Models](https://developers.deepgram.com/docs/tts-models). It wouldn't be in the voice of the original speaker, but it might get you what you need.
